### DISCUSSION TITLE
Flet app components

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [flet-dev/flet#994] [General]
Can flet app be broken down into components like navbar, sidebar

**Comment by iron3oxide** [comment]
Yes, you can find an example [here](https://github.com/iron3oxide/flet-mvp-counter/tree/master/mvp_counter).

**Reply by saurabhwadekar** [reply]
Thanks brother ❤️

**Reply by iron3oxide** [reply]
No problem. The repo is just an example project and thus the controls aren't very complex. In theory one could reuse a complex component like a menu across multiple pages/views though and even make each button do different things depending on the page/view.
